packages/jest/src/builders/jest/jest.impl.spec.ts

```diff
@@ -102,6 +103,47 @@ describe('Jest Builder', () => {
     );
   });
 
+  it('should send appropriate options to jestCLI when findRelatedTests is specified', async () => {
+    const run = await architect.scheduleBuilder('@nrwl/jest:jest', {
+      findRelatedTests: 'lib1.spec.ts,lib2.spec.ts',
```

Comment:
even though it doesn't really matter in this test, shouldn't we pass the list of source files instead of specs here?

Reply:
Yes. That's more realistic. I'll have that update in shortly.